.github/workflows/integrate.yaml

```diff
@@ -0,0 +1,63 @@
+# https://docs.github.com/en/actions
+
+name: "Integrate"
+
+on:
+  pull_request: null
+  push:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    name: "Build"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "8.0"
+
+        language:
+          - "de"
+          - "en"
+          - "es"
+          - "fr"
+          - "it"
+          - "ja"
+          - "pl"
+          - "pt_br"
+          - "ro"
+          - "ru"
+          - "tr"
+          - "uk"
+          - "zh"
```

Comment:
Are we missing any languages here?

Does it even make sense to build all languages here as well, or should we only use two languages, perhaps `de`, and `en`, to test building `en`, and also, to test the fallback to `en` when building a language other than `en`?

Reply:
Looks like these are all languages for which we have Git repos so far. Not sure about IT, PL and UK repos; these are inactive for the online docs.

I'm not sure whether doc-base changes need to be tested at all, since these are relatively rare, and issues should be caught pretty quickly when testing the languages. OTHO, doesn't hurt to have a full setup here as well.

Reply:
each language that is part of the ACTIVE_LANGUAGES array has a repo. Whether we want to deactivate them is a separate question IMO.